spark/v3.3/spark/src/main/scala/org/apache/spark/sql/execution/datasources/SparkExpressionConverter.scala

```diff
@@ -19,7 +19,6 @@
 
 package org.apache.spark.sql.execution.datasources
 
-import org.apache.iceberg.expressions.Expressions
```

Comment:
Is this un-intentional?

Reply:
it seems that we do not need this import statement. so i remove it here.